modules/install/pages/upgrade.adoc

```diff
@@ -76,19 +76,23 @@ For example, if you are upgrading three enterprise nodes (`*Node{nbsp}1*`, `*Nod
 | Starting Version |  Path to Current Version
 
 | 5.0.x
-| 5.0.x and 5.1.x -> 6.0.4+ -> 6.6 -> 7.2x
+| 5.0.x and 5.1.x -> 6.0.4+ -> 6.6 -> 7.2.x (except 7.2.3) -> 7.2.3
 
-5.0.x and 5.1.x -> 6.6 -> 7.2x
+5.0.x and 5.1.x -> 6.6 -> 7.2.x -> 7.2.x (except 7.2.3) -> 7.2.3
 
-5.5.0+ -> 6.6 -> 7.2x
+5.5.0+ -> 6.6 -> 7.2.x -> 7.2.x (except 7.2.3) -> 7.2.3
 
 | 6.x
-| 6.0 -> 6.5 -> 7.2x
+| 6.0 -> 6.6 -> 7.2.x -> 7.2.x (except 7.2.3) -> 7.2.3
 
 | 7.x
-| 7.0 -> 7.1 -> 7.2x
+| 7.0 -> 7.1 -> 7.2.3
+
+| 7.1.x
+| 7.2 -> 7.2.x -> 7.2.x (except 7.2.3) -> 7.2.3
```

Comment:
7.1.x is already covered by previous option (7.x) so I think this table row can be removed